CONTRIBUTING.md

```diff
@@ -25,3 +25,32 @@ Once ready for review, make sure to include a thorough PR description to help re
 We use the `git rebase` command to keep our commit history tidy.
 Rebasing is an easy way to make sure that each PR includes a series of clean commits with descriptive commit messages
 See [this tutorial](https://docs.gitlab.com/ee/topics/git/git_rebase.html) for a detailed explanation of `git rebase` and how you should use it to maintain a clean commit history.
+
+### Versioning
+
+When we need to fix bugs or introduce new features in our npm packages, we update the package version. To ensure consistency and simplicity in this process, we use a tool called [changesets](https://github.com/changesets/changesets).
+
+Each changeset contains three key pieces of information:
+* The package(s) to be published
+* Whether the release is major, minor, or a patch
```

Comment:
Might be worth linking to semver standards on major/minor/patch for folks who need a refresher.